pset/blinder.go

```diff
@@ -658,6 +642,123 @@ func (b *blinder) blindToken(index int, token, vbf, abf []byte) error {
 	return nil
 }
 
+func verifyBlinding(
```

Comment:
is it better to return error instead bool? we can propagate error so client has more insight on why blinding was not valid

Reply:
I think a bool is just enough. A transaction that is correctly blinded may be not valid in the sense that the proofs could be invalid, but it should be at least well-formed. This is not intended to be used to check that the blinding has been done correctly, but just to verify that the proofs are all ok.

Reply:
Problem is that the error would not be meaningful for the user. If wrong, the only solution is to retry. The blinding is either correct or not.